rewrite-java/src/main/java/org/openrewrite/java/tree/JavaType.java

```diff
@@ -156,8 +155,8 @@ public String toString() {
 
         private final String flyweightId;
 
-        private Class(String fullyQualifiedName,
-                      int flagsBitMap,
+        private Class(int flagsBitMap,
```

Comment:
Reordered the arguments in buildClass, buildMethod and the Variable constructors such that the flags are always the first argument.